docs/platforms/go/guides/gin/index.mdx

```diff
@@ -9,6 +9,16 @@ For a quick reference, there is a [complete example](https://github.com/getsentr
 
 ## Install
 
+<OnboardingOptionButtons
+  options={[
+    {
+      id: 'error-monitoring',
```

Comment:
Can we also simplify the 'error-monitoring' option? It will always be part (but passing 'error-monitoring' in the list is still worth it for explicity), but it will never be able to enable the checkbox. So configuring it in every instance of this component feels unnecessary

Reply:
Something like
```
<OnboardingOptionButtons
    'error-monitoring',
    'performance',
    'profiling',
  ]}
/>
```

Reply:
hardcode it as disabled?